sparksteps/__main__.py

```diff
@@ -185,6 +187,11 @@ def warn_and_override(args, argument, overrides, force_override=False):
     # Alias 'dynamic-pricing' -> 'dynamic-pricing-task'.
     args = warn_and_override(args, 'dynamic_pricing', ['dynamic_pricing_task'])
```

Comment:
Not specific to this PR, but if we're going to release a new version, I think that might also be a nice opportunity to remove the deprecation warnings here and to remove support for old command line arguments.